github/repos.go

```diff
@@ -538,6 +540,12 @@ type RequiredPullRequestReviews struct {
 	IncludeAdmins bool `json:"include_admins"`
```

Comment:
In order to fully resolve #606, I think you should mark `IncludeAdmins` as deprecated, or remove it.

Same for `IncludeAdmins` inside `RequiredStatusChecks` struct.

If you deprecate it, then change it to "Helps #606", so it won't be closed. We want to use that issue to track the entire change for https://developer.github.com/changes/2017-03-22-protected-branches-admin-improvements/, which includes removing the old way of setting admin.